dCommon/dConfig.cpp

```diff
@@ -10,16 +10,31 @@ dConfig::dConfig(const std::string& filepath) {
 	LoadConfig();
 }
 
+std::filesystem::path getConfigDir() {
```

Comment:
PascalCase for fns

Reply:
Also for private/local ones?

Reply:
Yeah, we do PascalCast across the board afaik.  Don't think i've seen an occurrence where we use camelCase for a function